src/types/promotion.ts

```diff
@@ -19,7 +21,9 @@ export interface Promotion {
   lon: number;
   liked: boolean;
   name: string;
+  placeId: string;
   promotionType: string;
+  restaurant: RestaurantDetails;
   restaurantName: string;
```

Comment:
What's `RestaurantDetails`?

Reply:
Oh it's the interface that Lily created for the return value from Google Places Details -- in #22 I think I refactor this to just be `Restaurant` which is a superset of the properties of `RestaurantDetails`